cirq-superstaq/cirq_superstaq/compiler_output_test.py

```diff
@@ -83,11 +83,96 @@ def test_compiler_output_repr() -> None:
         repr(css.compiler_output.CompilerOutput(circuit, qubit_map))
         == f"CompilerOutput({circuit!r}, {{}}, None, None, None, None)"
     )
+    assert (
+        css.compiler_output.CompilerOutput(circuit, qubit_map).__repr_pretty__()
+        == f"CompilerOutput({circuit!r}, {{}}, None, None, None, None)\n"
+    )
 
     circuits = [circuit, circuit]
     assert (
-        repr(css.compiler_output.CompilerOutput(circuits, [qubit_map]))
-        == f"CompilerOutput({circuits!r}, [{{}}], None, None, None, None)"
+        repr(css.compiler_output.CompilerOutput(circuits, [qubit_map, qubit_map]))
+        == f"CompilerOutput({circuits!r}, [{{}}, {{}}], None, None, None, None)"
+    )
+    assert css.compiler_output.CompilerOutput(
+        circuits, [qubit_map, qubit_map]
+    ).__repr_pretty__() == (
+        "CompilerOutput("
+        "\n    [cirq.Circuit(),  cirq.Circuit()],\n"
+        "    [{}, {}],\n"
+        "    None,\n"
+        "    None,\n"
+        "    None,\n"
+        "    None,\n"
+        ")\n"
+    )
+
+    circuits = [circuit] * 10
+    assert css.compiler_output.CompilerOutput(
+        circuits, [qubit_map, qubit_map]
+    ).__repr_pretty__() == (
+        "CompilerOutput(\n"
+        "    [\n"
+        f"        {circuits[0]!r},\n"
+        "        \n"
```

Comment:
is it possible/easy to avoid these spaces on black lines?

Reply:
Do you mean replace them with `\t` tab characters?

Reply:
oops sorry that was supposed to say "blank lines" - my thinking was just that there's no reason to include spaces in otherwise empty lines, e.g. this line could just be
```suggestion
        "\n"
```